### ISSUE TITLE
WX Python 4 builds: Getting stuck in sending keys mode

### ISSUE THREAD

**Issue by leonardder** [NVDARemote/NVDARemote#173]
When trying to control a remote system using recent WX Python 4 builds of NVDA next, NVDA Remote gets stuck in the Sending keys position. It is necessary to log out and back in  in order to get back control of the local system.

Related to nvaccess/nvda#7401

**Comment by leonardder** [comment]
cc @bramd

**Comment by bramd** [comment]
This is quite a blocker for this add-on given that the next NVDA release wil almost certainly have WXPython 4. After having a short look, it's not clear to me why this fails.
